app/src/main/java/com/github/onlynotesswent/model/folder/FolderRepository.kt

```diff
@@ -34,6 +34,15 @@ interface FolderRepository {
    */
   fun deleteFolderById(folderId: String, onSuccess: () -> Unit, onFailure: (Exception) -> Unit)
 
+  /**
+   * Deletes all folders from a user.
+   *
+   * @param userId The ID of the user to folders notes for.
+   * @param onSuccess Callback to be invoked if the notes are deleted successfully.
```

Comment:
I think this should be `Callback to be invoked if the folders are deleted successfully.`

Reply:
Yes thank you, fixed